cmd/marker/main.go

```diff
@@ -137,3 +112,68 @@ func keepAlive(healthCheckFile string, healthCheckInterval int) {
 
 	}, time.Duration(healthCheckInterval)*time.Second)
 }
+
+func parseOvsSocket(ovsSocket *string) string {
+	if *ovsSocket == "" {
+		glog.Fatal("ovs-socket must be set")
```

Comment:
suggesting to make this function to return an error and do the fatal in the main function. otherwise if someone will move this code to a different file they will probably forget the fatal and it will not be clear why the code crashing from a different file.

Reply:
> suggesting to make this function to return an error and do the fatal in the main function. otherwise if someone will move this code to a different file they will probably forget the fatal and it will not be clear why the code crashing from a different file.

Thanks Michael, uploaded a new patch with the your suggested changes.